### PR TITLE
Add cdt-gdb-adapter version for Theia 1.34

### PR DESCRIPTION
**Pull request by planger**
@jonahgraham Can you please confirm the versions I put in for cdt-gdb-adapter?

### CODE REVIEW

File: src/components/Releases.js [hunk 1]
```diff
@@ -107,10 +107,13 @@ const communityReleases = [
...
             },
             {
-                title: 'Eclipse CDT.cloud Debug Adapter',
+                title: 'Eclipse CDT Cloud Debug Adapter',
                 url: 'https://projects.eclipse.org/projects/ecd.cdt-cloud',
-                version: 'TBD',
-                modules: []
+                version: '0.0.93',
+                modules: [
+                    {modulename: 'cdt-gdb-vscode', url: 'https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode/0.0.93'},
```
Comment (jonahgraham) [inline src/components/Releases.js:114]: I just made a new version that pulls in all the latest and greatest. I expect it to be published overnight to https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode/0.0.95

Comment (jonahgraham) [reply src/components/Releases.js:114]:
```suggestion
                    {modulename: 'cdt-gdb-vscode', url: 'https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode/0.0.95'},
```

**Comment by planger** [comment]
@jonahgraham Thanks for the releases and the updates! I've applied your suggestion.
@JonasHelming This change should be now good to go.

**Review by JonasHelming — APPROVED** [review]

**Comment by JonasHelming** [comment]
@jonahgraham can you aprove?

**Review by jonahgraham — APPROVED** [review]

**Pull request merged**